usvm-ts/src/test/resources/ir/MinValue.json

```diff
@@ -0,0 +1,636 @@
+{
```

Comment:
Its ok for now, but I don't want to have these jsons in the repository in the main branch. It should be replaced with jacodb automatic solution